README.rst

```diff
@@ -1 +1,150 @@
-Stub readme.
+edx_event_bus_kafka
+=============================
+
+|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge|
+
+The ``README.rst`` file should start with a brief description of the repository,
+which sets it in the context of other repositories under the ``edx``
+organization. It should make clear where this fits in to the overall edX
+codebase.
+
+Kafka implementation for Open edX event bus.
+
+Overview (please modify)
+------------------------
+This repository details an implementation of the event bus using Kafka.
+
+Essentially what the event bus does is it processes events from microservices, events can be sent in to the event bus and sent out to be consumed by consumers.
```

Comment:
No need for "Essentially". Also I wouldn't use "processes" here, since the event bus doesn't exactly process events, it more queues them up for other services to process.
```suggestion
The event bus acts as a broker between services publishing events and other services that consume these events.
```